Gruntfile.js

```diff
@@ -1,7 +1,40 @@
-module.exports = function(grunt) {
+
+module.exports = grunt => {
+
+  require('time-grunt')(grunt);
 
   grunt.initConfig({
 
+    clean: {
+      build: {
+        src: ['.tmp/*']
+      }
+    },
+
+    compass: {
+      dist: {
+        options: {
+          sassDir: 'styles',
```

Comment:
To make sure you are getting all the css files use: styles/**/*.[sass, scss]
Remember that ** will include all folders, and that is important